pinot-spi/src/main/java/org/apache/pinot/spi/trace/DefaultRequestContext.java

```diff
@@ -402,6 +419,136 @@ public void setReduceTimeMillis(long reduceTimeMillis) {
     _reduceTimeMillis = reduceTimeMillis;
   }
 
+  @Override
+  public long getNumConsumingSegmentsQueried() {
+    return _numConsumingSegmentsQueried;
+  }
+
+  @Override
+  public void setNumConsumingSegmentsQueried(long numConsumingSegmentsQueried) {
```

Comment:
(nit) I think we can use int instead of long here. May not really matter in the grand scheme of things but off late we have seen a lot of heap usage improvement opportunities from production heap dumps. So just something to consider

Reply:
this long is from `BrokerResponse`